### PR TITLE
Frontdesk

### PR DESCRIPTION
**Pull request by menchb**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -17,6 +19,14 @@
     return view('welcome');
 });
 
-Auth::routes();
+Route::get('/dashboard', function () {
```
Comment (Teds30) [inline routes/web.php]: '/frontdesk/dashboard'

File: routes/web.php [hunk 2]
```diff
@@ -17,6 +19,14 @@
...
+    return view('frontdesk/dashboard');
+});
+
+Route::get('/information', function () {
```
Comment (Teds30) [inline routes/web.php]: '/frontdesk/information'

**Comment by FN-FAL113** [comment]
discord test

**Review by Teds30 — APPROVED** [review]
goods

**Review by FN-FAL113 — APPROVED** [review]

**Pull request merged**